src/util.h

```diff
@@ -221,6 +227,32 @@ static SECP256K1_INLINE void secp256k1_memczero(void *s, size_t len, int flag) {
     }
 }
 
+/* Cleanses memory to prevent leaking sensitive info. Won't be optimized out. */
+static SECP256K1_INLINE void secp256k1_memclear(void *ptr, size_t len) {
+#if defined(_MSC_VER)
+    /* SecureZeroMemory is guaranteed not to be optimized out by MSVC. */
+    SecureZeroMemory(ptr, len);
```

Comment:
I looked at the same code in Core again, and found this PR, and commented there: https://github.com/bitcoin/bitcoin/pull/26950#issuecomment-2429613882

Reply:
Everything else being equal, keeping the code in sync with Core is nice, but as I said there, I have a slight preference to stick to the current `#if defined(_MSC_VER)` because we have no OS-specific macros anywhere else in the library (ignoring the examples). See also https://github.com/bitcoin-core/secp256k1/pull/1625.